### DISCUSSION TITLE
Thanks for creating this :)

### DISCUSSION THREAD

**Discussion by denismack** [prasathmani/tinyfilemanager#503] [Ideas]
I really love this script and use it a lot in production, but with a "Recycle Bin" this would be perfect for recovering files deleted by mistake.

Pls, consider this in the future ... 

Great job guys!

**Comment by prasathmani** [comment]
Thank you for using tinyfilemanager. 
as tinyfilemanger running on single file without database, so it very difficult to track the deleted files, but still you can have option in server/os level, check this https://stackoverflow.com/a/28374470/1164642

**Reply by denismack** [reply]
Perfectly, but I think it would be possible to implement a simple move to a trash folder, instead of unlink, it would be a false delete.
And a real unlink just in cleaning the "trash folder".
